### PR TITLE
Add flycheck-erlang-library-path variable

### PR DESCRIPTION
**Pull request by yzprofile**

### CODE REVIEW

**Comment by lunaryorn** [comment]
@yzprofile What is the different between the include and the library path?

File: flycheck.el [hunk 1]
```diff
@@ -5863,6 +5863,16 @@ Relative paths are relative to the file being checked."
   :safe #'flycheck-string-list-p
   :package-version '(flycheck . "0.24"))
 
+(flycheck-def-option-var flycheck-erlang-library-path nil erlang
+  "A list of library directories for Erlang.
+
+The value of this variable is a list of strings, where each
+string is a directory to add to the include path of erlc.
```
Comment (lunaryorn) [inline flycheck.el]: This docstring does not appear to be correct.  Isn't it for the library path?

**Comment by lunaryorn** [comment]
@yzprofile And of course, many thanks for this contribution.  I'll definitely merge it, but please address my remarks.

Thanks!

**Comment by yzprofile** [comment]
Fixed, Sorry for that.

**Comment by lunaryorn** [comment]
@yzprofile It's all fine, thanks for this contribution!

**Pull request merged**